core/runtime/binaryen/runtime_environment.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_RUNTIME_BINARYEN_RUNTIME_ENVIRONMENT
+#define KAGOME_CORE_RUNTIME_BINARYEN_RUNTIME_ENVIRONMENT
+
+#include <memory>
+
+#include <boost/optional.hpp>
+
+#include "outcome/outcome.hpp"
+
+namespace kagome::common {
+  class Buffer;
+}
```

Comment:
You use **namespace { class }** instead **using** to avoid including additional headers? =)

Reply:
I use it to avoid including headers that are unnecessary here, that's right, but, honesly, I don't know how can I **use** **using** for this.

Reply:
If definition was included, we can doing just `use kagome::common::Buffer;` as usual. But sometimes is bad way (e.g., in case recursive definition). Right here is okay, I was just interested.